Build broken with GCC7

- OS version: e.g. SLC6
 - Compiler version: e.g GCC 7.2
 - Package version: master
 - Reproduced by: make
 - Output: 
```
[ 49%] Building CXX object source/LumiCalReco/CMakeFiles/LumiCalReco.dir/src/MarlinLumiCalClusterer.cpp.o
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp: In member function 'void MarlinLumiCalClusterer::CreateClusters(std::map<int, std::map<int, ClusterClass*> >&, EVENT::LCEvent*)':
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:300:28: error: 'placeholders' is not a namespace-name
       using namespace std::placeholders;  //_1, _2
                            ^~~~~~~~~~~~
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:300:40: error: expected namespace-name before ';' token
       using namespace std::placeholders;  //_1, _2
                                        ^
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:305:65: error: 'bind' is not a member of 'std'
         sort(mcParticlesVec.begin(), mcParticlesVec.end(), std::bind(PositionCompAsc, xs, ys, _1, _2));
                                                                 ^~~~
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:305:65: note: suggested alternative: 'find'
         sort(mcParticlesVec.begin(), mcParticlesVec.end(), std::bind(PositionCompAsc, xs, ys, _1, _2));
                                                                 ^~~~
                                                                 find
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:305:95: error: '_1' was not declared in this scope
         sort(mcParticlesVec.begin(), mcParticlesVec.end(), std::bind(PositionCompAsc, xs, ys, _1, _2));
                                                                                               ^~
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:305:95: note: suggested alternative: 'y1'
         sort(mcParticlesVec.begin(), mcParticlesVec.end(), std::bind(PositionCompAsc, xs, ys, _1, _2));
                                                                                               ^~
                                                                                               y1
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:305:99: error: '_2' was not declared in this scope
         sort(mcParticlesVec.begin(), mcParticlesVec.end(), std::bind(PositionCompAsc, xs, ys, _1, _2));
                                                                                                   ^~
/cvmfs/clicdp.cern.ch/iLCSoft/builds/test/x86_64-slc6-gcc7-opt/FCalClusterer/HEAD/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp:305:99: note: suggested alternative: '._92'
         sort(mcParticlesVec.begin(), mcParticlesVec.end(), std::bind(PositionCompAsc, xs, ys, _1, _2));
                                                                                                   ^~
                                                                                                   ._92
make[2]: *** [source/LumiCalReco/CMakeFiles/LumiCalReco.dir/src/MarlinLumiCalClusterer.cpp.o] Error 1
make[1]: *** [source/LumiCalReco/CMakeFiles/LumiCalReco.dir/all] Error 2
make: *** [all] Error 2
```

--- a/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp
+++ b/source/LumiCalReco/src/MarlinLumiCalClusterer.cpp
@@ -19,6 +19,7 @@
 #include <map>
 #include <vector>
 #include <iomanip>
+#include <functional>
 
 /* >> */
 

